main_exercise10.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+from exercise10 import (
+    get_price_quantity,
+    get_subtotal,
+    calculate_sum_of_subtotals,
+    calculate_total_tax_for_items,
+    calculate_grand_total,
+)
+
+TAX_RATE = 5.5
+
+def main():
+    """Create a self-checkout system that calculates the grand total for three items"""
+
+    price_item1, quantity_item1 = get_price_quantity()
+    print("\n")
```

Comment:
I think in Python3, there's a shortcut. You can just use `print()` to print a new line.